consensus/hotstuff/sig_validator.go

```diff
@@ -0,0 +1,26 @@
+package hotstuff
+
+import (
+	"github.com/onflow/flow-go/consensus/hotstuff/model"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// SigValidator only validate a message's signature part.
+// It transforms the vote or block into a message, and uses the verifier, which doesn't have
+// the knowledge of vote/block, to verify the signature.
+type SigValidator interface {
```

Comment:
The difference between `consensus/hotstuff/SigValidator` and `consensus/hotstuff/Validator` is that the former validates the signature only, whereas the later validates everything except signature.